README.md

```diff
@@ -167,6 +171,32 @@ or the following to your [Emacs init file][]:
 
 The socket server REPL configuration options are described [here](https://dev.clojure.org/display/design/Socket+Server+REPL).
 
+#### Lumo Socket REPL
```

Comment:
Can this section follow the same format as the other three? For coherence I guess. We also don't need to tell here about the devtools I guess. People should be already aware of that..